client.go

```diff
@@ -703,12 +703,16 @@ func (c *Client) Deploy(ctx context.Context, path string) (err error) {
 	}
 
 	// Deploy a new or Update the previously-deployed Function
-	c.progressListener.Increment("Deploying function to the cluster")
-	result, err := c.deployer.Deploy(ctx, f)
-	if result.Status == Deployed {
-		c.progressListener.Increment(fmt.Sprintf("Function deployed at URL: %v", result.URL))
-	} else if result.Status == Updated {
-		c.progressListener.Increment(fmt.Sprintf("Function updated at URL: %v", result.URL))
+	if !dryrun {
+		c.progressListener.Increment("Deploying function to the cluster")
```

Comment:
why would you skip the progress message ? I would suggest to do something like for kubectl:

```
$ kubectl run  --image quay.io/minio/mc mc --dry-run=client
pod/mc created (dry run)
```

i.e. print out the progress message but also indicate that it's just a dry run.

Reply:
btw, `kubectl run` follows my reasoning in the review comment: It does *not* print out the Pod manifest's yaml but the _steps_ it would perform (and includes validation and that it can reach the server).